src/components/Circles/ReactionsToolbar.test.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import { ReactionsToolbar } from './ReactionsToolbar';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { GraphQLClientContextProvider } from '../../hooks/useGraphQLClient';
+import { ActivePost } from '../../hooks/useInfinitePosts';
+import {
+  useCreateReactionMutation,
+  useUndoReactionMutation,
+} from '../../hooks/useReactionMutations';
+import { useUser } from '../../hooks/useUser';
+
+jest.useFakeTimers();
+jest.mock('../../hooks/useReactionMutations', () => ({
+  useCreateReactionMutation: jest.fn(),
+  useUndoReactionMutation: jest.fn(),
+}));
+jest.mock('../../hooks/useUser', () => ({
+  useUser: jest.fn(),
+}));
+
+const useCreateReactionMutationMock = useCreateReactionMutation as jest.Mock;
+const useUndoReactionMutationMock = useUndoReactionMutation as jest.Mock;
+const useUserMock = useUser as jest.Mock;
+useUserMock.mockReturnValue({
+  data: {
+    id: 'userId',
+  },
+});
+
+const baseURL = 'https://some-domain/unit-test';
+const toolbarComponent = (post: ActivePost) => (
+  <QueryClientProvider client={new QueryClient()}>
+    <GraphQLClientContextProvider baseURL={baseURL} />
```

Comment:
Should this wrap `ReactionsToolbar`?

Reply:
I would have thought so but it seems to be working as expected 🤔 I'll take a look at that and address if needed in another PR.